fhir-operation-document/src/main/java/com/ibm/fhir/operation/document/DocumentOperation.java

```diff
@@ -61,7 +61,7 @@ protected Parameters doInvoke(FHIROperationContext operationContext, Class<? ext
             
             composition = (Composition) resource;
             
-            Bundle bundle = buildDocument(operationContext, composition, resourceHelper);            
+            Bundle bundle = buildDocument(operationContext, composition, resourceHelper);
```

Comment:
it's weird to find that spaces are added from time to time to the files, my eclipse has save action to remove the spaces at the line end, but sometime when I edit the same file again, I can still see spaces at the same file... really strange ...

Reply:
Yeah, I just clean it up when I see it.  Probably should have left out of a big PR like this, but too much energy to dance around it once I've made the local change.